authors/apps/authentication/views.py

```diff
@@ -207,9 +207,10 @@ def put(self, request, **kwargs):
         except (TypeError, ValueError, OverflowError, User.DoesNotExist):
             return Response(
                 {"response": 'A user with this email was not found.'})
-        if not default_token_generator.check_token(user, token):
-            return Response({"response": 'Invalid reset tokens'},
-                            status=status.HTTP_401_UNAUTHORIZED)
+        is_valid = default_token_generator.check_token(user, token)
+        if not is_valid:
+            return Response({"response": 'Token has expired'},
+                            status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Using an expired token means you're unauthorized. The status code should be `401`